### PR TITLE
Contact Journal Day Screen Accessibility (EXPOSUREAPP-5134)

### PR DESCRIPTION
**Pull request by axelherbstreith**
Small PR that changes the announcement of the Floating Action Button in the contact journal day screen when exploring by touch. 
Changed from "Ort" to "Ort hinzufügen" e.g. "Person" to "Person hinzufügen"

To test the changes, activate TalkBack and navigate to the day screen of the contact journal. Explore Floating Action Button by touch.

### CODE REVIEW

**Review by kolyaopahle — APPROVED** [review]
LGTM 👍 talk back worked on pixel 5 (Android 11)

**Review by kolyaopahle — DISMISSED** [review]
LGTM 👍 talk back worked on pixel 5 (Android 11)

File: Corona-Warn-App/src/main/res/values/contact_diary_strings.xml [hunk 1]
```diff
@@ -141,6 +141,11 @@
     <!-- XTXT: Edit (description for screen readers) -->
     <string name="accessibility_edit">"Edit"</string>
 
+    <!-- XBUT: Add Person Button (description for screen readers) -->
```
Comment (LukasLechnerDev) [inline Corona-Warn-App/src/main/res/values/contact_diary_strings.xml:144]: Should be`<!-- XACT: xxx ->` ?

**Review by LukasLechnerDev — DISMISSED** [review]
LGTM, tested on Samsung S20+, Android 11.

One small remark.

**Review by LukasLechnerDev — APPROVED** [review]

**Review by BMItr — APPROVED** [review]

**Review by janetback — COMMENTED** [review]
UA approved

**Review by janetback — APPROVED** [review]
UA reviewed

**Pull request merged**